server/plugin/client.go

```diff
@@ -273,6 +275,14 @@ func (c *client) UpdatePipelineApprovalRequest(pipelineApproveRequestPayload *se
 	return statusCode, err
 }
 
+func (c *client) UpdatePipelineRunApprovalRequest(pipelineApproveRequestPayload []*serializers.PipelineApproveRequest, organization, projectID, mattermostUserID string) (*serializers.PipelineRunApproveResponse, int, error) {
+	pipelineApproveRunRequestURL := fmt.Sprintf(constants.PipelineRunApproveRequest, organization, projectID)
+
+	var pipelineRunApproveResponse *serializers.PipelineRunApproveResponse
+	_, statusCode, err := c.CallJSON(c.plugin.getConfiguration().AzureDevopsAPIBaseURL, pipelineApproveRunRequestURL, http.MethodPatch, mattermostUserID, &pipelineApproveRequestPayload, &pipelineRunApproveResponse, nil)
+	return pipelineRunApproveResponse, statusCode, err
```

Comment:
shouldn't we return nil here if err is present as a good practice?

Reply:
that was handled but yes considering the practise we should do that